org-roam.el

```diff
@@ -608,33 +608,35 @@ is the `org-roam-node'."
                                      (propertize alias 'invisible t))
                              node)))))
 
+(defcustom org-roam-node-annotation-function #'org-roam-node--annotation
+  "The function used to return annotations in the minibuffer for Org-roam nodes.
+This function takes a single argument NODE, which is an `org-roam-node' construct.")
+
 (defun org-roam-node-read (&optional initial-input filter-fn require-match)
   "Read and return an `org-roam-node'.
 INITIAL-INPUT is the initial prompt value.
 FILTER-FN is a function applied to the completion list.
 If REQUIRE-MATCH, require returning a match."
   (let* ((nodes (org-roam-node--completions))
          (nodes (funcall (or filter-fn #'identity) nodes))
-         (node (completing-read "Node: "
-                                (lambda (string pred action)
-                                  (if (eq action 'metadata)
-                                      '(metadata
-                                        (annotation-function . org-roam-node--annotation)
-                                        (category . org-roam-node))
-                                    (complete-with-action action nodes string pred)))
-                                nil require-match initial-input)))
+         (node (completing-read
+                "Node: "
+                (lambda (string pred action)
+                  (if (eq action 'metadata)
+                      '(metadata
+                        (annotation-function . (lambda (title)
+                                                 (funcall org-roam-node-annotation-function
+                                                          (get-text-property 0 'node node-title))))
```

Comment:
looks like there's an issue with this change:
`(org-roam-node-find)` **Error in post-command-hook (selectrum--update): (void-variable node-title)**

Reply:
fixed, thanks